frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -1088,18 +1087,26 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
                 value,
             })
         },
-        reportActivationSideBarShown: ({ activeTasksCount, completedTasksCount, completionPercent }) => {
-            posthog.capture('activation sidebar shown', {
-                active_tasks_count: activeTasksCount,
-                completed_tasks_count: completedTasksCount,
-                completion_percent_count: completionPercent,
+        reportActivationSideBarOpened: ({ tab }) => {
+            posthog.capture('activation sidebar opened', {
+                tab,
+            })
+        },
+        reportActivationSideBarClosed: ({ tab }) => {
+            posthog.capture('activation sidebar closed', {
+                tab,
```

Comment:
This is fine to put here, and you correctly found the pattern. BUT a while ago we kind of stopped doing this because this file was unnecessarily bloated and it's not really necessary to have all these in one file. So in the future you can just put these `posthog.capture` calls in the logic where they are actually called. Simplifies things.